Quansight/conda-store
# #431 and #432: `conda-lock` cannot install `conda-store` created lockfiles

Fixes #431: adds a column to `CondaPackageBuild` table to track the `package.package_tarball_full_path` extension so we can build the lockfile package url(s) for the correct tarball

Fixes #432: adds a space 

### Manual Testing

0. Create a successful build with the following spec:

```
name: scratch
channels:
- conda-forge
dependencies:
- nspr=4.33=h295c915_0
```
1. ssh into your local `conda-store-server` or `conda-store-worker` container (doesn't matter since they both have `conda-lock`)
2. download the related build lockfile
3. make sure you can successfully `conda-lock install <lockfile-path> -n scratch`

### Async Data Migration Script:

> There really should be an async celery migration script to backfill the new column `tarball_ext` for past packages/builds that have artifacts registered. Might/might not be worth it. Looking into how hard that is now

giving up on this ☝️
